lang/java/reef-io/src/main/java/org/apache/reef/io/network/group/impl/task/OperatorTopologyStructImpl.java

```diff
@@ -325,6 +326,49 @@ public void sendToChildren(final byte[] data, final ReefNetworkGroupCommProtos.G
     return retVal;
   }
 
+  @Override
+  public byte[] recvFromChildren() {
```

Comment:
Sure. I'll extract the `EMPTY_BYTE_ARR` checking part as a separate function and make `recvFromParent()` and `recvFromChildren()` use that function.